.travis.yml

```diff
@@ -3,11 +3,13 @@ language: python
 python:
     - "2.7"
     - "3.3"
+    - "3.4"
+    - "3.5"
 
 install:
-    - pip install pytest-cov --use-mirrors
-    - pip install pytest-pep8 --use-mirrors
-    - pip install coveralls --use-mirrors
+    - pip install pytest-cov
+    - pip install pytest-pep8
+    - pip install coveralls
```

Comment:
.travis.yml was broken after a pip update on TravisCI. It's fixed in #79, please merge the latest changes from master to resolve.